app/scripts/controllers/add-events.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (Api, $scope, $routeParams, $location) {
   Api.Patients.get({ id: $routeParams.patientId }, success, error)
 
   $scope.events = [
-    { code: 'sample-event', display: 'Sample Event', formName: 'FormBuilderaddEventSampleEvent' }
+    { code: 'birth', display: 'Birth Registration', formName: 'FormBuilderAddEventBirth' }
```

Comment:
Thanks @rcrichton thats a very good explanation of the differences!, So for now, We are ONLY dealing with Birth Notification?